### ISSUE TITLE
Post 1.1 update errors

### ISSUE THREAD

**Issue by jhalbrecht** [Windows-XAML/Template10#234]
FWIW I just updated the template vsix extension from 1.0 to 1.1 and the NuGet from 1.0.4 to 1.0.5. I got the following errors in an existing project. Searched the repository for 'release notes' didn't find anything specifically helpful. 

In my project the Template10 library, via nuget, is upgraded, but the template is based on the installed version at the time of the project creation correct? That is if I began the app with Template10 1.0 vsix installed the .xaml is not updated in my existing project when I update the Template10 extension?

![image](https://cloud.githubusercontent.com/assets/502208/10527970/4ab24638-7348-11e5-884e-14efc95ff166.png)

(Hmm trying to put a 'question' label on this...)

**Comment by jhalbrecht** [comment]
I added Custom.xaml from the new template, and removed the old Custom.[Dark|Light].xaml files from the project.Works!

Hey the new styling sure looks a lot better to me. Nice.

Are there any other places in the .xaml I should consider updating from the new version 1.1?

Is this a worthy of an entry into some release notes, or upgrade notes? Or is it just so obvious I should hang my head in shame for suggesting it?

**Comment by JerryNixon** [comment]
Those are the breaking changes.

**Comment by jhalbrecht** [comment]
If there is a breaking changes document for the installed templates from 1.0 to 1.1 you could also add the DataContext on the Grid in the About PivotItem of the SettingsPage.xaml created by the Template10 Minimal template.

V1.0

```
 <PivotItem Header="About">
                <Grid>
                    <Border HorizontalAlignment="Right"
[...]
```

Fix in 1.1

```
 <PivotItem Header="About">
                <Grid DataContext="{Binding AboutPartViewModel}">
                    <Border HorizontalAlignment="Right"
[...]
```
